### DISCUSSION TITLE
how to use database transaction in FastAPI

### DISCUSSION THREAD

**Discussion by maxwelltesla** [fastapi/fastapi#8916] [Questions]
###how to use database transaction in FastAPI
* [ ] Thanks FastAPI
* [ ] As we all know the database transaction has specition function, but when I want to use it in my fullstack,  cannot find an example to study .
can anyone show a example or a link here?
thanks.

**Comment by teamhide** [comment]
I made boilerplate that has sqlalchemy transaction feature.

https://github.com/teamhide/fastapi-boilerplate

**Comment by maxwelltesla** [comment]
thanks  a  lot

**Comment by FIRDOUS-BHAT** [comment]
I'm looking for it in tortoise-orm
